server/src/test/resources/csars/yaml/valid/lamp-csar/template.yaml

```diff
@@ -0,0 +1,102 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+description: Template for deploying a LAMP stack.
+metadata:
+  template_name: lamp-stack-template
+  template_author: stupro-toscana
+  template_version: 1.0
+
+
+topology_template:
+  inputs:
+    database_name:
+      type: string
+    database_user:
+      type: string
+    database_password:
+      type: string
+    database_port:
+      type: integer
+    my_mysql_rootpw:
+      type: string
+    my_mysql_port:
+      type: integer
+
+  node_templates:
+    my_app:
+      type: tosca.nodes.WebApplication
+      requirements:
+        - host: apache_web_server
+        - database_endpoint: my_db
+      interfaces:
+        Standard:
+          create: scripts/install_php.sh
+          start:
+            implementation:
+              primary: scripts/start_myphpapp.sh
+              dependencies:
+                - scripts/myphpapp.php
+            inputs:
+              db_host: { get_attribute: [ apache_web_server, private_address ] }
+              db_name: { get_property: [ my_db, name ] }
+              db_user: { get_property: [ my_db, user ] }
+              db_password: { get_property: [ my_db, password ] }
+              db_port: { get_property: [ SELF, database_endpoint, port ] }
+
+    apache_web_server:
+      type: tosca.nodes.WebServer
```

Comment:
Why do you not use the WebServer.Apache type?

Reply:
for me and @mmunozba there was a reason, but unfortunately i cant remember it.
it should work with this normative type as well but you are right there is a non-normative type webserver.apache. @mmunozba, do you remember why we used the normative type?

Reply:
The CSAR uses the WebServer.Apache type now.